.circleci/config.yml

```
@@ -145,6 +259,20 @@ workflows:
           filters:
             branches:
               ignore: main
+      - webpack:
+          name: webpack-deploy
+          requires:
+            - npm-install-deploy
+          filters:
+            tags:
+              only: /.*/
+      - compile-translations:
+          name: compile-translations-deploy
+          requires:
+            - pip-install-deploy
+          filters:
+            tags:
+              only: /.*/
```

```suggestion
      - webpack:
          requires:
            - npm-install
      - compile-translations:
          requires:
            - pip-install
```
Here, you copied the wrong parts of the integreat-cms config: The `*-deploy` jobs are part of the `deploy` workflow, not the `develop` workflow.